### PR TITLE
Add adjustOpacity method to Styles

### PR DESCRIPTION
**Pull request by jmophoto**
Add adjustOpacity method to Styles constants.

This takes hex color and opacity params and returns an rbga string.

To use
```
<div style={{ backgroundColor: Styles.Colors.ORANGE }}></div>
<div style={{ backgroundColor: Styles.adjustOpacity(Styles.Colors.ORANGE, 0.1) }}></div>
```
screen shot 2016-04-01 at 4 39 11 pm

### CODE REVIEW

**Comment by cerinman** [comment]
Looks good.  :shipit:

File: src/constants/Style.js [hunk 1]
```diff
@@ -81,5 +81,14 @@ module.exports = {
     }
 
     return (usePound ? '#' : '') + (g | (b << 8) | (r << 16)).toString(16);
+  },
+
+
+  adjustOpacity (color, opacity) {
```
Comment (bsbeeks) [inline src/constants/Style.js]: I'd name this `adjustHexOpacity` since it's strictly dealing with hex values.

**Comment by jmophoto** [comment]
@bsbeeks: agreed. Pushed up those changes so this will need another ship it if all looks good.

**Comment by cerinman** [comment]
:shipit:

**Pull request merged**